ext/bin/pdbbox-init

```diff
@@ -176,4 +176,40 @@ ssl-key = $abs_pdbbox/ssl/pdb.key.pem
 disable-update-checking = true
 EOF
 
+# do some tuning on PostgreSQL
+
+# generally accepted configuration parameters:
+# Set the shared_buffers to be 1/4 of the amount of RAM
+
+# for development only (considered unsafe in production):
+# Turn off fsync
+# Turn off synchronous commit
+
+case "$OSTYPE" in
+    darwin*)
+        total_ram_bytes=$(sysctl hw.memsize | awk '{ print $2 }') 
+        total_ram_kb=$(expr $total_ram_bytes / 1024) ;;
+    *)
+        total_ram_kb=$(cat /proc/meminfo | grep MemTotal: | awk '{ print $2 }') 
+esac
+
+total_ram_mb=$(expr $total_ram_kb / 1024)
+# these values come from pgtune and the postgresql tuning wiki
+shared_buffers=$(expr $total_ram_mb / 4)
+maintenance_work_mem=$(expr $total_ram_mb / 15)
+
+cat >> "$pdbbox/pg/data/postgresql.conf" <<EOF
+
+# tuning parameters
+fsync = off
```

Comment:
I think I'm inclined to leave this off and just make sure we suggest using `pg_ctl start -F ...` in our docs, so that the default is still safe.